Makefile

```diff
@@ -305,6 +302,32 @@ run-torture-verilator: verilate
 check-torture:
 	grep 'All signatures match for $(test-location)' $(riscv-torture-dir)/$(test-location).log
 	diff -s $(riscv-torture-dir)/$(test-location).spike.sig $(riscv-torture-dir)/$(test-location).rtlsim.sig
+vcs_command := vcs -full64 -sverilog -v2k_generate -timescale=1ns/1ps -debug_access +lint=TFIPC-L +lint=PCWM       \
+                    $(filter-out %.vhd, $(ariane_pkg))                                     \
+                    $(filter-out src/fpu_wrap.sv, $(filter-out %.vhd, $(src)))             \
+                    +define+$(defines)                                                     \
+                    src/util/sram.sv                                                       \
+                    +incdir+src/axi_node                                                   \
+                    -CFLAGS "-std=c++11 -I../tb/dpi"                                       \
+                    $(list_incdir)                                                         \
+                    tb/ariane_tb.sv tb/dpi/SimJTAG.cc tb/dpi/remote_bitbang.cc	           \
+
+#		    /opt/Xilinx/Vivado/2018.1/data/verilog/src/glbl.v                      \
```

Comment:
Thanks, that looks really good. Note that I would suggest that you base your stuff on the `ariane_next` branch. We already merged some fpga changes in there and you are likely better off using this branch (as we are going to merge all the fpga stuff in the upcoming week).